### DISCUSSION TITLE
Need a Assistance from your side to fulfill my requirement

### DISCUSSION THREAD

**Discussion by nazerhussain86** [JaidedAI/EasyOCR#1270] [General]
my requirement is to process invoice pdf file 
what ever readable or scanned pdf 
I need to extract text line by line and need specific info like invoice number , invoice date , shipper address and description of product .
So Need a Assistance from your side to fulfill my requirement

**Comment by CharlyJazz** [comment]
Hey let me help you:

You can achieve this with the next roadmap:

1. Use cv to extract all the rentangle width large contours
2. Use easeyocr on the lang that u need to predict the text
3. Send all the information to chatgpt and ask for the invoice analysis on json format output using chat completation.

Vote up if u like it
